backend/endpoints/signUp/signUp.test.ts

```diff
@@ -0,0 +1,45 @@
+import { v4 as uuid } from "uuid";
+import { expect } from 'chai'
+import axios from "axios";
+
+import { authFrontend, firebaseSignUp } from "../../helpers/firebaseFrontend";
+import UserClient from "../../dynamo/UserClient";
+import { User } from "../../types/User";
+import { auth } from "../../helpers/firebase";
+
+describe('signUp', function () {
+    let userId: string;
+    let email: string;
+    let token: string;
+    
+    beforeEach(async function () {
+        email = `test${uuid()}@testing.com`;
+        await firebaseSignUp(email, "pppppp");
+        token = await authFrontend.currentUser?.getIdToken()!;
+        const decodedToken = await auth.verifyIdToken(token);
+        const { uid } = decodedToken;
+        userId = uid;
+    })
+
+    afterEach(async function () {
+        await auth.deleteUser(userId)
+    })
+
+    it('should create a user with firebase before adding them to aws database', async function () {
```

Comment:
The lambda isn't responsible for creating the user in firebase, right? 
```suggestion
    it('add a firebase user to aws database', async function () {
```